Cairo Desktop/Cairo Desktop/SupportingClasses/RegistryEventWatcher.cs

```diff
@@ -0,0 +1,57 @@
+﻿using CairoDesktop.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management;
+using System.Security.Principal;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CairoDesktop.SupportingClasses
+{
+    public sealed class RegistryEventWatcher : IDisposable
+    {
+        private const string REGISTRY_HIVE = @"HKEY_USERS";
+        private const string REGISTRY_VALUE = @"AccentColorMenu";
+        private const string REGISTRY_KEY = @"SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Explorer\\Accent";
+
+        private readonly ManagementEventWatcher _watcher;
+        private readonly CairoApplicationThemeService _themeService;
+
+        public RegistryEventWatcher(CairoApplicationThemeService ts)
+        {
+            WqlEventQuery query = new WqlEventQuery(GetQuery());
+            _watcher = new ManagementEventWatcher(query);
+            _themeService = ts;
+            WatchRegistry(true);
```

Comment:
Ideally the constructor wouldnt call `WatchRegistry`.

Reply:
Added StartWatcher() and StopWatcher()